WidgetExtension/Variants/FollowersCount/FollowersCountWidget.swift

```diff
@@ -0,0 +1,149 @@
+        loadCurrentEntry(for: configuration, in: context) { entry in
+            completion(Timeline(entries: [entry], policy: .after(.now)))
+        }
+    }
+}
+
+struct FollowersCountEntry: TimelineEntry {
+    let date: Date
+    let account: FollowersEntryAccountable?
+    let configuration: FollowersCountIntent
+    
+    static var placeholder: Self {
+        FollowersCountEntry(
+            date: .now,
+            account: FollowersEntryAccount(
+                followersCount: 99_900,
+                displayNameWithFallback: "Mastodon",
+                acct: "mastodon",
+                avatarImage: UIImage(named: "missingAvatar")!,
+                domain: "mastodon"
+            ),
+            configuration: FollowersCountIntent()
+        )
+    }
+    
+    static var unconfigured: Self {
+        FollowersCountEntry(
+            date: .now,
+            account: nil,
+            configuration: FollowersCountIntent()
+        )
+    }
+}
+
+struct FollowersCountWidget: Widget {
+    private var availableFamilies: [WidgetFamily] {
+        if #available(iOS 16, *) {
+            return [.systemSmall, .accessoryRectangular, .accessoryCircular]
+        }
+        return [.systemSmall]
+    }
+
+    var body: some WidgetConfiguration {
+        IntentConfiguration(kind: "Followers", intent: FollowersCountIntent.self, provider: FollowersCountWidgetProvider()) { entry in
+            FollowersCountWidgetView(entry: entry)
+        }
+        .configurationDisplayName("Followers")
+        .description("Show number of followers.")
+        .supportedFamilies(availableFamilies)
+    }
+}
+
+private extension FollowersCountWidgetProvider {
+    func loadCurrentEntry(for configuration: FollowersCountIntent, in context: Context, completion: @escaping (FollowersCountEntry) -> Void) {
+        Task {
+            guard
+                let authBox = WidgetExtension.appContext
+                    .authenticationService
+                    .mastodonAuthenticationBoxes
+                    .first
+            else {
+                guard !context.isPreview else {
+                    return completion(.placeholder)
+                }
+                return completion(.unconfigured)
+            }
+            
+            guard let desiredAccount: String = {
+                guard let account = configuration.account else {
+                    return authBox.authenticationRecord.object(in: WidgetExtension.appContext.managedObjectContext)?.user.acct
+                }
+                return account
+            }() else {
+                return completion(.unconfigured)
+            }
+            
+            let resultingAccount = try await WidgetExtension.appContext
+                .apiService
+                .search(query: .init(q: desiredAccount, type: .accounts), authenticationBox: authBox)
+                .value
+                .accounts
+                .first!
```

Comment:
Is it safe to assume there's always a `first`? Asking for a crash :)

Reply:
Hm good catch, might be a left over from my first tries, I'll fix this.